tests/tck/features/admin/Sessions.feature

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2021 vesoft inc. All rights reserved.
+#
+# This source code is licensed under Apache 2.0 License.
+Feature: Test sessions
+
+  Background:
+    Given a nebulacluster with 3 graphd and 1 metad and 1 storaged
+
+  Scenario: Show sessions
+    When executing query:
+      """
+      SHOW SESSIONS;
+      """
+    Then the result should contain:
+      | SessionId | UserName | SpaceName | CreateTime | UpdateTime | GraphAddr | Timezone | ClientIp           |
+      | /\d+/     | "root"   | ""        | /.*/       | /.*/       | /.*/      | 0        | "::ffff:127.0.0.1" |
+    When executing query:
+      """
+      CREATE USER user1 WITH PASSWORD 'nebula1';
+      CREATE SPACE s1(vid_type=int);
+      USE s1;
+      """
+    Then the execution should be successful
+    And wait 1 seconds
+    When login graphd[1] with user1 and nebula1
```

Comment:
login "graphd[1" with "user" and "nebula1"

Reply:
meaningful